samples/core/visualization/tensorboard_gcs.py

```diff
@@ -70,4 +70,4 @@ def my_pipeline(
     log_dir=f'gs://{{kfp-default-bucket}}/tensorboard/logs/{dsl.RUN_ID_PLACEHOLDER}'
 ):
     prepare_tb_task = prepare_tensorboard(log_dir_uri=log_dir)
-    tensorboard_task = train_op(log_dir=prepare_tb_task.outputs['log_dir_uri'],)
+    tensorboard_task = train_op(log_dir=log_dir).after(prepare_tb_task)
```

Comment:
Also not ideal. so maybe add a comment above to revert this change once we fixed the sanitization issue?

Reply:
Maybe I missed anything, what's the problem with this line?

Reply:
The version before the change is clear and explicit dependency -- The output from one task is consumed by the second task.
After the change, this dependency becomes indirect. Technically speaking, all artifact can be passed this way. But I don't think this is a recommended pattern. My concern is, without knowing the reason why this is written this way, user may start copy this pattern.

Reply:
Note the prepare_tensorboard task does not write anything to the URI. (Previous logic is that it gets an URI as input and output the same URI, which is unnecessary anyway)

Reply:
I see. Thanks for the explanation.